src/components/Buildbaron/index.ts

```diff
@@ -0,0 +1,3 @@
+import { loadable } from "components/SpruceLoader";
+
+export const BuildBaron = loadable(() => import("./BuildBaron"));
```

Comment:
nice.

Reply:
thanks! It's technically part of the Task page, and not used by any other pages, so should this be moved to /pages/Task instead?